src/test/java/org/folio/rest/impl/EntitiesCrudTest.java

```diff
@@ -38,11 +39,19 @@
   @EnumSource(TestEntities.class)
   void testVerifyCollection(TestEntities testEntity) throws MalformedURLException {
     logger.info(String.format("--- mod-data-export %s test: Verifying database's initial state ... ", testEntity.name()));
-    getRequest(testEntity.getEndpoint()).then()
-      .log()
-      .all()
-      .statusCode(200)
-      .body("totalRecords", equalTo(0));
+    if (testEntity.getClazz().getName().equals(MappingProfile.class.getName())) {
```

Comment:
Though this is not a good way to do it, I think this will change once we add default job profile too, so we can ignore for this PR

Reply:
I planned to move it back in PR about job profile, but ok, i will ignore them